rome-osgi-test/pom.xml

```diff
@@ -11,6 +10,43 @@
     </parent>
 
     <artifactId>rome-osgi-test</artifactId>
+    <packaging>jar</packaging>
```

Comment:
Packaging is jar by default. Also, for this particular artifact we don't even care about packaging, because we don't deploy it.